README.md

```diff
@@ -277,6 +283,76 @@ _Why Rebase?_, It's a great to make a clean looking tree, it appears as if you n
 
 **[⬆ back to top](#overview)**
 
+## Tricks
+
+### Stripspace
+
+- Removes trailing whitespace
+- Collapse New lines
+- Adds new lines to end of file
+
+  - Simply pass in the file that you want to strip
+
+  ```bash
+  $ git stripspace < text_file.txt
+  ```
+
+### Previous Branch
+
+- Quickly go back to your last branch
+
+  ```bash
+  $ git checkout - 
+  ```
+
+### Clean Status
+
+- Add `-sb` to get a cleaner `git status`
+
+  ```bash
+  $ git status -sb
+  ```
+
+### Auto Correct
+
+- Set auto correct to true for Git commands
+
+  ```bash
+  $ git config --global help.autocorrect 1
+  ```
+
+- Results in:
+
+  ```bash
+  $ git comit -m "commit message"
+  # WARNING: You called a Git command named 'comit', which does not exist.
+  # Continuing under the assumption that you meant 'commit'
+  # in 0.1 seconds automatically...
+  ```
+
+### Alias
+
+- Add shortcuts to reduce your typing
+
+  ```bash
+  $ git config --global alias.alias_command git_command
+  ```
+
+Examples:
+
+- Set `git stat` to `git status -sb`
+
+  ```bash
+  $ git config --global alias.stat status -sb
```

Comment:
I'm pretty sure you have to wrap `status -sb` in quotations (" ") in order for it to group the option in with the alias